src/swoop/api/routers/processes.py

```diff
@@ -28,22 +26,20 @@ class Params(BaseModel):
     description: str | None
     handler: str | None
     argo_template: str | None = None
-    cache_enabled: bool | None = None
 
 
-def to_process_summary(workflowConfig: list[dict]) -> Process:
+def to_process_summary(workflowConfig: Workflow) -> Process:
     return ProcessSummary(
-        id=workflowConfig["name"],
-        title=workflowConfig["name"],
-        name=workflowConfig["name"],
-        processID=workflowConfig["name"],
-        version=workflowConfig["version"],
-        description=workflowConfig.get("description"),
-        handler=workflowConfig.get("handler"),
-        argoTemplate=workflowConfig.get("argo_template"),
-        cacheEnabled=workflowConfig.get("cache_enabled"),
-        cacheKeyHashIncludes=workflowConfig.get("cache_key_hash_includes"),
-        cacheKeyHashExcludes=workflowConfig.get("cache_key_hash_excludes"),
+        id=workflowConfig.name,
+        title=workflowConfig.name,
+        name=workflowConfig.name,
+        processID=workflowConfig.name,
+        version=workflowConfig.version,
+        description=workflowConfig.description,
+        handler=workflowConfig.handler,
+        # argoTemplate=workflowConfig.workflow_template,
```

Comment:
The current submodel-specific fields `argo_template` and `sfn_arn` I don't think need to be exposed to end users.